pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -1042,6 +1042,17 @@ func ensureSystemCgroups(rootCgroupPath string, manager cgroups.Manager) error {
 				}
 
 				finalErr = fmt.Errorf("failed to move PID %d into the system container %q: %v", pid, name, err)
+			} else {
+				err := manager.Set(nil)
+				if err != nil {
+					name := ""
+					cgroups, err := manager.GetCgroups()
+					if err == nil {
+						name = cgroups.Name
+					}
+
+					finalErr = fmt.Errorf("failed to set system container %q: %v", name, err)
+				}
```

Comment:
As far as I understand the code here, it is merely used for moving processes from the root cgroup into a specified one, that has no resource limits configured. In other words, cgroup manager is only used to put processes into a cgroup, not to set any resources.

Therefore (and if I am reading this right), there is no need to call `Set` here. Also, there is no need to set Resources in `func createManager`.